views/welcome.html

```diff
@@ -83,8 +84,22 @@ <h2>Integrations</h2>
           </div>
         </div>
       </div>
-      
-      
+
+
+      <div class="tab-pane fade show" id="list-create-azure-host-policy" role="tabpanel" aria-labelledby="list-create-azure-host-policy-list">
+        <div class="row">
+          <div class="col-6">
+            <Br/>
+            <h2>Create Azure Host Policy</h2>
+            Generate the Host policy needed to authenticate using Azure Authenticator from Azure token<br/>
```

Comment:
I would go with "Generate a policy defining a host that can authenticate with the Azure Authenticator, using an Azure AD token"